repose-aggregator/src/docs/asciidoc/recipes/docker.adoc

```diff
@@ -110,13 +101,9 @@ Let's break that command down and take a closer look at what it is doing:
     See our http://www.openrepose.org/versions/latest/release-notes.html[Release Notes] page for a list of available versions.
 <3> This option defines the tag to apply to the image.
     It is in `repository:tag` format.
-<4> The last parameter is the directory that contains the Dockerfile.
-    The Dockerfile itself must exist as a child of the provided directory.
-+
-[NOTE]
-====
-This directory will also be treated as the Docker build context.
-====
+<4> The last parameter is the path of the `docker build` context.
```

Comment:
I can't say why, exactly,  but `path to the` feels more correct than `path of the` to me.